maven/pom-core.xml

```diff
@@ -12,6 +12,32 @@
   <description>Thin Java abstractions for the standard built-in objects for Javascript.</description>
   <url>https://www.gwtproject.org</url>
 
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <scm>
+    <connection>scm:git:git@github.com:google/elemental2.git</connection>
```

Comment:
Fwiw, I like to use the HTTPS URL here, as it just works without any credentials (even though people can also do the translation easily).

Reply:
Seems reasonable.